cmd/ttn-lw-cli/commands/users.go

```diff
@@ -415,6 +451,33 @@ var (
 			return nil
 		},
 	}
+	usersTransferRightsCommand = &cobra.Command{
+		Use:     "transfer-rights [user-id] [receiver-id]",
+		Aliases: []string{"transfer", "move", "rights-move"},
```

Comment:
Let's not add these aliases. Perhaps in the future we want to `transfer` or `move` users somewhere else, which would then give us conflicts.